extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcBuildTimeConfig.java

```diff
@@ -6,27 +6,39 @@
 import io.quarkus.runtime.annotations.ConfigItem;
 
 @ConfigGroup
-public class DataSourceBuildTimeConfig {
+public class DataSourceJdbcBuildTimeConfig {
+
+    /**
+     * If we create a JDBC datasource for this datasource.
+     */
+    @ConfigItem(name = ConfigItem.PARENT, defaultValue = "true")
+    public boolean enabled = true;
 
     /**
      * The datasource driver class name
      */
     @ConfigItem
-    public Optional<String> driver;
+    public Optional<String> driver = Optional.empty();
 
     /**
      * Whether we want to use regular JDBC transactions, XA, or disable all transactional capabilities.
      * <p>
      * When enabling XA you will need a driver implementing {@link javax.sql.XADataSource}.
      */
     @ConfigItem(defaultValue = "enabled")
-    public TransactionIntegration transactions;
+    public TransactionIntegration transactions = TransactionIntegration.ENABLED;
 
     /**
      * Enable datasource metrics collection. If unspecified, collecting metrics will be enabled by default if the
      * smallrye-metrics extension is active.
      */
     @ConfigItem
-    public Optional<Boolean> enableMetrics;
+    public Optional<Boolean> enableMetrics = Optional.empty();
+
+    /**
+     * The datasource driver short name
+     */
+    @ConfigItem
+    public Optional<String> shortName;
```

Comment:
is that to be used as `....jdbc.shortname = h2` ? along side `kind`? I am not following.

Reply:
Ah, this shouldn't be there, it's a leftover. Let me fix that.

Reply:
Fixed.